src/core/Text/Paragraph/ParagraphText.tsx

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import { useStyles } from './styles';
+import Typography from '@material-ui/core/Typography';
+
+type Type = 'small' | undefined;
+
+interface ParagraphTextProps {
```

Comment:
The same as above. 
* We don't need to add `Text` at the end since its coming from `Text/Something` so its automatically understandable.
* We need to Omit variant and any other API from Typography that is not required since we are providing our own and extend the default interface here with our own.
* type should be variant instead.
* children is auto inferred
* case undefined should be default
*